<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1667191297346' stop='1667191303195'>
    <name>Create Appointment</name>
    <title>Create Appointment</title>
    <test-cases>
        <test-case start='1667191297347' status='broken' stop='1667191303195'>
            <name>Create Appointment successfully</name>
            <title>Create Appointment successfully</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='wdio'/>
                <label name='thread' value='0-0'/>
                <label name='language' value='javascript'/>
                <label name='framework' value='wdio'/>
                <label name='thread' value='0-0'/>
            </labels>
            <parameters>
                <parameter kind='argument' name='browser' value='chrome-106.0.5249.119'/>
                <parameter kind='argument' name='browser' value='chrome-106.0.5249.119'/>
            </parameters>
            <steps/>
            <attachments/>
            <failure>
                <message>CreateAppointment is not defined</message>
                <stack-trace>ReferenceError: CreateAppointment is not defined
                    at Context.&lt;anonymous> (C:\WebdriverIO\src\test-scripts\/Testcase_1.js:21:9)
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>